pysma/const.py

```diff
@@ -111,6 +111,12 @@
     SUNIT: "A",
     SFACTOR: 1000,
 }
+SENSOR_CURRENT_TOTAL = { # Example: "6100_00664F00":{"7":[{"val":3501}]} # 10038=GridMs.TotA\6704896\Grid current
```

Comment:
To prevent long lines, please put comments above each sensor instead of on the same line.
Not sure if the example and values from SBFspot have any added value.

Please also format file using `black`.